pkgs/top-level/php-packages.nix

```
@@ -310,7 +310,7 @@ lib.makeScope pkgs.newScope (self: with self; {
 
     phalcon = callPackage ../development/php-packages/phalcon { };
 
-    php-spx = callPackage ../development/php-packages/php-spx { };
+    php-spx = lib.warn "php-spx is deprecated, use spx instead" (callPackage ../development/php-packages/spx { });
```

```suggestion
    php-spx = lib.warn "php-spx is deprecated, use spx instead" self.extensions.spx;
```
I think you can reuse the derivation instead of calling it again. Can you try?